src/pages/auth/login/Login.jsx

```diff
@@ -1,8 +1,57 @@
 import React from 'react'
-
+import Navbar from '../../components/navbar/Navbar'
+import '../login/login.css'
+import axios from 'axios'
+import { useState } from 'react'
+import { useNavigate } from "react-router-dom";
```

Comment:
You can move all the library imports to the top. just a preference. most f the OSS projects follows this.

Reply:
noted, this is something I didn't know. will definitely add this in next version across files, thanks for the suggestion.